rasa_core/tracker_store.py

```diff
@@ -31,6 +31,20 @@ def __init__(self, domain, event_broker=None):
         self.domain = domain
         self.event_broker = event_broker
 
+    @staticmethod
+    def find_tracker_store(domain, store=None):
+        if store is None or store.store_type is None:
```

Comment:
`if None in {store, store.store_type}`

Reply:
I think this one is controversial, I'd argue the `or` way is easier to understand

Reply:
OK then, leave it out